### ISSUE TITLE
What is your project?

### ISSUE THREAD

**Issue by gparmer** [gwu-iot/20_mystery-men#1]
You didn't meet the last deadline. You need to meet the next. I need project details.

Perhaps something that integrates Sam's interest in photography with IoT?

**Comment by samfrey99** [comment]
Apologies. We completed a plan in a separate repository and overlooked the requirement to submit it through the form. I've added the document here. It is still pretty vague at this point. Our primary goal is to create a door lock controlled by a Raspberry Pi that can be unlocked via facial recognition. I already have one of GWs Raspberry Pis from last summer along with the camera and necessary cables. Alvaro has begun looking into appropriate APIs for facial recognition on Azure since he has experience there. We've been looking at our project in a few levels we'd like to achieve:

1. Facial recognition requires a button press to be triggered. Upon recognition, the Pi actuates a DC motor that opens the lock. 
2. Facial recognition is automatic. It seems like this would require a continuous stream of image data to the cloud portion of the project which may present speed or cost restrictions.
3. A mobile application interface to add new people to the recognition list or unlock the door remotely without relying on facial recognition at all.

Unfortunately, I don't think these line up too nicely with the checkpoints. Reaching our "level 1" within a week and a half doesn't seem feasible, but solid progress towards that goal (ie. actuating the lock with the Pi,  transmitting image data to the cloud component) is far more achievable. If "level 2" does in fact present speed or cost restrictions that we cannot overcome, our current plan is to move forward with "level 3" and focus on creating a more usable device for the real world because there's no use for a door lock that doesn't let you in if you're wearing a hat. 

Once again, my apologies for missing the deadline. We would greatly appreciate any feedback you have on our project idea.
